tests/test_agent.py

```diff
@@ -50,6 +50,7 @@ def test_set_state():
 
 
 def test_change_state():
+    np.random.seed()
```

Comment:
It is generally a good idea to set back to random number generator for cleanup at the end of your test if you do this.

For getting the state of random generator you can use `state = np.random.get_state()` and then at the end of the test you can set it back using `np.random.set_state(state)`.

Reply:
Doesn't `np.random.seed()` just read a "random" number from `/dev/urandom`?  I think you need to provide a seed, like `np.random.seed(0)`.

Reply:
Yes, I agree. It needs to have some number to be consistent. By default, it just takes current timestamp as the seed.

Reply:
@erikwijmans is right:

https://docs.scipy.org/doc/numpy-1.15.1/reference/generated/numpy.random.RandomState.html

> Random seed used to initialize the pseudo-random number generator. Can be any integer between 0 and 2**32 - 1 inclusive, an array (or other sequence) of such integers, or None (the default). If seed is None, then RandomState will try to read data from /dev/urandom (or the Windows analogue) if available or seed from the clock otherwise.